packages/renderer/src/lib/help/HelpActions.svelte

```diff
@@ -0,0 +1,88 @@
+<script lang="ts">
+import { faDiscord, faSlack } from '@fortawesome/free-brands-svg-icons';
+import {
+  faComment,
+  faExclamationTriangle,
+  faExternalLinkAlt,
+  faLightbulb,
+  faListUl,
+} from '@fortawesome/free-solid-svg-icons';
+import { DropdownMenu } from '@podman-desktop/ui-svelte';
+
+import HelpMenu from './HelpMenu.svelte';
+
+let showMenu = false;
+let outsideWindow: HTMLDivElement;
+window.events?.receive('toggle-help-menu', toggleMenu);
+
+function toggleMenu(): void {
+  showMenu = !showMenu;
+}
+
+function handleEscape({ key }: KeyboardEvent): void {
+  if (key === 'Escape') {
+    showMenu = false;
+  }
+}
+
+function onWindowClick(e: any): void {
+  showMenu = outsideWindow.contains(e.target);
+}
+
+function createOnClick(onClick: () => void): () => void {
+  return () => {
+    toggleMenu();
+    onClick();
+  };
+}
+
+function createOpenExternal(link: string) {
+  return createOnClick(() => {
+    window.openExternal(link).catch(console.error);
+  });
+}
+</script>ß
```

Comment:
```suggestion
</script>
```

Weird letter ?

![image](https://github.com/user-attachments/assets/354eb803-2c54-4db1-b0e1-c5440f6fffcf)